packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -291,9 +291,12 @@ export const Tooltip = styled.span< TooltipProps >`
 	user-select: none;
 	line-height: 1.4;
 
+	@media not ( prefers-reduced-motion ) {
+		transition: opacity 120ms ease;
```

Comment:
Should we remove `transition: opacity 120ms ease;` from L290 then?

Reply:
Oops, great catch!